lib/v8-log-to-ticks.js

```diff
@@ -15,7 +15,28 @@ module.exports = wrapper
 
 async function wrapper (isolateLogPath, node) {
   const filtered = await filterLinesThatAreTooLong(isolateLogPath)
-  return v8LogToTicks(filtered, node)
+  const normalised = await normalizeWindowsLog(filtered)
+  return v8LogToTicks(normalised, node)
+}
+
+async function normalizeWindowsLog (isolateLogPath) {
+  if (process.platform !== 'win32') return isolateLogPath
+
+  // Only needed because of a V8 tick processor bug that
+  // treats C:(\A...) as an escape sequence when parsing the csv file
+
+  const dest = isolateLogPath + '.normalised'
+  let data = await readFile(isolateLogPath, 'utf8')
+  data = data.split('\n').map(escapePath).join('\n')
+  await writeFile(dest, data)
+  return dest
+}
+
+function escapePath (line) {
+  const startPath = line.indexOf(':\\')
+  if (startPath === -1) return line
+  // \u005c is the unicode escape for the \ char
+  return line.slice(0, startPath) + line.slice(startPath).replace(/\\/g, '\\u005c')
```

Comment:
If the V8 fix is in, would this code break?

Reply:
Potentially yes, depending on how the fix lands (my current fork just disables all unescaping in their parser, which *would* break this).

Once that lands we have time to fix it here tho, so I think this is okay for now.

Reply:
Actually only replacing [^\]\ with the escape sequence properly future proofs a lot better :+1: